package/scie-pants.lift.json

```diff
@@ -99,6 +99,18 @@
               "{scie.bindings.configure:PANTS_SHA_FIND_LINKS}"
             ]
           },
+          "pants-maybe-native-client": {
+            "description": "Runs a hermetic Pants installation via the native client (falling back to the legacy client if the native client is not available).",
+            "env": {
+              "=PANTS_VERSION": "{scie.bindings.configure:PANTS_VERSION}",
+              "PANTS_BUILDROOT_OVERRIDE": "{scie.bindings.configure:PANTS_BUILDROOT_OVERRIDE}",
+              "=_PANTS_SERVER_EXE": "{scie.bindings.install:VIRTUAL_ENV}/bin/pants"
+            },
+            "exe": "{scie.bindings.install:MAYBE_NATIVE_CLIENT_BINARY}",
```

Comment:
Does the native client now handle starting pantsd when it's not already up?

Reply:
Yea: see https://github.com/pantsbuild/pants/pull/18957.